pkg/distro/defs/rhel-9/distro.yaml

```diff
@@ -1182,3 +1182,33 @@ image_types:
                 *edge_commit_x86_64_pkgset
               aarch64:
                 *edge_commit_aarch64_pkgset
+
+  "azure-cvm":
+    package_sets:
+      os:
+        - include:
+            - "@minimal-environment"
+            - "chrony"
+            - "cloud-init"
+            - "cloud-utils-growpart"
+            - "cryptsetup"
+            - "NetworkManager-cloud-setup"
+            - "openssh-server"
+            - "redhat-cloud-client-configuration"
+            - "redhat-release"
+            - "tpm2-tools"
+            - "WALinuxAgent"
+        - exclude:
+            - "dracut-config-rescue"
+            - "iwl*"
+            - "kernel"
+            - "kernel-core"
+            - "kernel-modules"
+            - "linux-firmware*"
+
+            - "grub2-common"
+            - "grub2-efi-x64"
+            - "grub2-tools"
+            - "grub2-tools-minimal"
```

Comment:
I would expect some (if not all) of these to come from the `platform` instead of being defined here.

Reply:
It would make sense to try to remove them from the excludes, since after the other commits in the PR, `grub2` should not be pulled in IIRC...

Reply:
I'll move them around and see which ones are necessary to keep.

Reply:
I droppped most of the excludes but kept them in the image type package sets.  One issue is that the firmware exclusion is distro-specific right now, so we can't just put them all in the platform.  I'm not entirely sure if the rest belong in the platform now.